mrp_workcenter_group/security/model_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+<!-- Multi - Company Rules -->
+
+<record model="ir.rule" id="workcenter_group_comp_rule">
+    <field name="name">Workcenter multi-company</field>
+    <field name="model_id" ref="model_workcenter_group"/>
+    <field name="global" eval="True"/>
+    <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+</record>
```

Comment:
For my own education, isn't this already auto-generated, do we have to do it on all our new models?